packages/zosfiles/src/api/methods/get/Get.ts

```diff
@@ -46,6 +46,14 @@ export class Get {
             reqHeaders = [ZosmfHeaders.X_IBM_BINARY];
         }
 
+        if (options.encoding) {
```

Comment:
Is this a different behavior when compared to Upload.ts? Imagine binary and encoding are both specified. In Upload.ts, the text;fileEncoding header is never added because it is in an else block. Here, the encoding overrides the binary header.

I would suggest making the if an else if to make it consistent.